ethicml/algorithms/inprocess/blind.py

```diff
@@ -0,0 +1,32 @@
+"""Simply returns a random (but legal) label."""
+
+import numpy as np
+import pandas as pd
+
+from ethicml.common import implements
+from ethicml.utility import DataTuple, Prediction, TestTuple
+
+from .in_algorithm import InAlgorithm
+
+__all__ = ["Blind"]
+
+
+class Blind(InAlgorithm):
+    """Returns a random label."""
+
+    def __init__(self, seed: int = 888) -> None:
+        """Init Majority."""
+        super().__init__(name="Blind", is_fairness_algo=False)
+        self.seed = seed
+
+    @implements(InAlgorithm)
+    def run(self, train: DataTuple, test: TestTuple) -> Prediction:
+        train_y_vals = train.y.drop_duplicates()
+
+        self.random = np.random.RandomState(self.seed)
```

Comment:
ah, if you do it like this, you don't need the `self.`.

my idea was to put this line into the `__init__` (then you would need to assign the random state to `self`).

not sure which way is better. do you want subsequent runs to have the same random state? I don't know.

Reply:
I was thinking about this, because I don't think it's ever really been discussed before. There are 2 sources of randomness, that I can tell - the data, and the model. 

When we run a cross validation, for example, I think that only the data should change. But, if we're running multiple repeats, I'm not sure if we should just change the data, or change the data and the model randomness together. 

I don't think there's a case where we just want randomness in the model... (that I can think of)

Reply:
> a case where we just want randomness in the model

uncertainty estimates? :D

Reply:
@MylesBartlett do you have an opinion on this?

Reply:
Thinking about it more, you're right that it doesn't make sense that subsequent runs have different seeds. So, leave it as it is.

Reply:
> > a case where we just want randomness in the model
> 
> uncertainty estimates? :D

Yes, but I think that this should probably be handled by the model itself